### ISSUE TITLE
Accessibility best practices

### ISSUE THREAD

**Issue by xiwenc** [mxlint/mxlint-cli#31]
Notes from meeting with Sophie:

Accessibility best practices
- Alt-text required for all images
- keyboard navigation: tab-index should not be used unless needed explicitly (Sophie)
- heading usage: H2 always after H1, H3 after H2, etc...
- heading usage: only one H1 heading per form (opinionated)

UI Testing:
- text readability contrast of text
- screen reader compatibility

**Comment by bartzantingh** [comment]
I am not sure this can be done on a level that is needed for WCAG compatibility. A number of things can probably be checked, like the alt texts for images, but already there is additional complexity there: not every image needs an alt text (e.g. when it is flagged as 'decorative'). On top of that, the accessibility features in Mendix Studio are, in my opinion, lackluster in the best cases, straight out awful in the worst cases, and very dependent on using widgets like the Accessibility Helper.
